base_layer/core/src/base_node/sync/header_sync/validator.rs

```diff
@@ -309,16 +309,16 @@ mod test {
 
         #[tokio::test]
         async fn it_fails_if_height_is_not_serial() {
-            let (mut validator, _, tip) = setup_with_headers(2).await;
+            let (mut validator, _, tip) = setup_with_headers(12).await;
             validator.initialize_state(tip.hash()).await.unwrap();
             let mut next = BlockHeader::from_previous(tip.header());
-            next.height = 10;
+            next.height = 14;
             let err = validator.validate(next).unwrap_err();
             unpack_enum!(BlockHeaderSyncError::ValidationFailed(val_err) = err);
             unpack_enum!(ValidationError::BlockHeaderError(header_err) = val_err);
             unpack_enum!(BlockHeaderValidationError::InvalidHeight { actual, expected } = header_err);
-            assert_eq!(actual, 10);
-            assert_eq!(expected, 3);
+            assert_eq!(actual, 14);
+            assert_eq!(expected, 13);
```

Comment:
Why did these values require changing? The offset is still the same we just brought the actual and expected closer together, but I don't get why.

Reply:
So the problem what happened here is that we had 2 blocks in the chain. 
Now the validator took in the latest height, aka 10. And then said, if the header height is ten, we need 9 previous header heights to check the median timestamp.  So it was failing on not enough header timestamps. 
This test we want it to fail on incorrect header height. 

The solution was to make the chain 12, which is the higher than max median timestamps required. Then we can bump of the next header height above 13 to fail on the correct error.

Reply:
The fact that they are now closer together is got nothing to do with the values, we just need the initial to be higher than 11,
and then the next header to be higher than the expected
aka its expecting 13 in this case, so we make it 14